commons/clever-cloud-overview/common-application-configuration.md

```diff
@@ -70,3 +70,23 @@ repository. The script file **MUST** have execution right set.
 
 In the example above, the script is a file named `myScript` in the `clevercloud` folder.
 That is, right beside the `<instance_type>.json` file.
+
+## Workers
+
+You can run background tasks during in parallel of your application. They will be restarted automatically on error.
+
+The workers automatically inherit your application's environment and are launch in the directory in which the
```

Comment:
The workers run in the same environment as your application's.
They are launched in the application's directory